manifests/ca.pp

```diff
@@ -92,7 +92,7 @@
 
       exec { "generate dh param ${name}":
         command  => '. ./vars && ./clean-all && ./build-dh',
-        timeout  => 1800,
+        timeout  => 20000,
```

Comment:
I am seeing that is EasyRSA version is 3, then `timeout` is not present.
Can you add same `timeout` in [the other exec generate dh](https://github.com/voxpupuli/puppet-openvpn/blob/52fd0d61dce2539c775420348cf4ba5ea2a5c518/manifests/ca.pp#L167)  also ?